### PR TITLE
Create initial Open_Source_LLM and Smart_Vector_Index_DB Tool Documents

### PR DESCRIPTION
**Pull request by gjwoods**
# Description

Add the tool documents for 2 new\in progress Prompt Flow Tools, the Open_Source_LLM and the Smart_Vector_Index_DB tool.

# All Promptflow Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](../CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [X] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/reference/tools-reference/open_source_llm-tool.md
Comment (wangchao1230) [inline docs/reference/tools-reference/open_source_llm-tool.md:1]: please include your md in index.md file

File: docs/reference/tools-reference/open_source_llm-tool.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Open SourceLLM
```
Comment (thy09) [inline docs/reference/tools-reference/open_source_llm-tool.md:1]: Maybe we can have two different PRs for the two different tools?

Comment (gjwoods) [reply docs/reference/tools-reference/open_source_llm-tool.md:1]: Good point - here are the new PR's:
#370
#371

File: docs/reference/tools-reference/open_source_llm-tool.md
Comment (gjwoods) [reply docs/reference/tools-reference/open_source_llm-tool.md:1]: Thank you for the guidance. I have updated it in the split PR:
#370 - Create tools reference doc for the Smart Vector Index DB tool.
#371 - Create tools reference doc for the Open_Source_LLM tool.

**Pull request closed without merging**